deps/rabbitmq_mqtt/src/rabbit_mqtt_processor.erl

```diff
@@ -819,7 +835,7 @@ ensure_queue(Qos, #proc_state{ channels      = {Channel, _},
                                    %%
                                    %% see rabbitmq/rabbitmq-mqtt#37
                                    auto_delete = CleanSess,
-                                   arguments   = Qos1Args },
+                                   arguments   = maybe_quorum(Qos1Args, CleanSess, QueueQ1)},
```

Comment:
It can but that's not safe. Combining quorum queues with clean session set to `true` doesn't make much sense, this
was our key decision when we discussed this feature with @Gsantomaggio in mid-February IIRC.

Reply:
Thanks for the explanation.